src/components/Home.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchRegions } from '../redux/region/regionSlice';
+import './Home.css';
+
```

Comment:
- [optional] nice job. According to the guidelines and also to make your application easy to use, you were to ensure the Home page has a means of filtering the list of categories using given parameters, e.g. category name.
- [optional] please, your home page has a back arrow which leads to nowhere. Kindly fix that. The back arrows are for the details page, leading back to the home page. Nice work.

![Screenshot_23](https://github.com/balatstar/metrics-webapp/assets/65298048/35d70fc3-f58f-4b02-9ebf-07acd05bb83a)